config/crds/bootstrap.go

```diff
@@ -176,6 +177,16 @@ func CreateSingle(ctx context.Context, client apiextensionsv1client.CustomResour
 	})
 }
 
+// Unmarshal YAML-decodes the give embedded file name into the target.
+func Unmarshal(fileName string, o interface{}) error {
```

Comment:
nit: whenever you are writing non-typesafe code it should not be done lightly. in my personal opinion, the two stdlib calls in this function are not complex enough to warrant a helper that exits the type system

Reply:
I can change the signature to accept a concrete type.